src/commands/anoncreds/issuer.rs

```diff
@@ -199,9 +205,10 @@ impl IssuerCommandExecutor {
             .map_err(map_err_trace!())
             .map_err(|err| CommonError::InvalidStructure(format!("Invalid claim_req_json: {}", err.to_string())))?;
 
-        let claim_def_uuid = self.wallet_service.get(wallet_handle, &format!("seq_no::{}", &claim_req_json.claim_def_seq_no))?;
-        let claim_def_json = self.wallet_service.get(wallet_handle, &format!("claim_definition::{}", &claim_def_uuid))?;
-        let claim_def_private_json = self.wallet_service.get(wallet_handle, &format!("claim_definition_private::{}", &claim_def_uuid))?;
+        let claim_def_id = claim_req_json.issuer_did.clone() + ":" + &claim_req_json.schema_seq_no.to_string();
```

Comment:
Looks like `claim_def_id ` is used in multiple places. Can we have a helper method for this?